src/partials/footer-content.hbs

```diff
@@ -73,7 +73,7 @@
         </section>
         <section class="footer-bottom">
             <div class="footer-inside">
-                <p class="footer-copyright"><span>&copy;2021</span> MuleSoft LLC, a Salesforce company</p>
+                <p class="footer-copyright"><span>&copy;1</span> MuleSoft LLC, a Salesforce company</p>
```

Comment:
How does `&copy;1` turn into the 2022 copyright?

Reply:
```suggestion
                <p class="footer-copyright"><span>&copy;2022</span> MuleSoft LLC, a Salesforce company</p>
```